.github/workflows/platform-ci.yml

```diff
@@ -75,8 +81,8 @@ jobs:
           else
             pip install virtualenv flaky
             if [ -n "$WITH_KUBERNETES" ]; then
-              ./ci/install-minikube.sh
               pip install kubernetes
+              kubectl get pods -A
```

Comment:
Is this statement a health check?

Reply:
Yeah. List all pods to check the state of minikube.